content/comm/website/adding-a-chapter/index.md

```diff
@@ -0,0 +1,66 @@
+## Create a new file
+Create a new file in the [data/chapters/](https://github.com/rladies/website/blob/main/data/chapters) folder by [using this link](https://github.com/rladies/website/new/main/?filename=data/chapters/country-state-city.json&value=%5B%0A%20%20%7B%0A%20%20%20%20%22urlname%22%3A%20%22rladies-%22%2C%20%20%20%20%20%20%20%20//meetup%20link%0A%20%20%20%20%22status%22%3A%20%22%22%2C%20%20%20%20%20//%20prospective%2C%20active%20or%20inactive%0A%20%20%20%20%22country%22%3A%20%22%22%2C%20%20%20%20%20%20%20%20%20%20//%20country%2C%20capitalised%0A%20%20%20%20%22state.region%22%3A%20%22%22%2C%20%20%20%20//%20state%20or%20reigion%2C%20capitalised%2C%20optional%0A%20%20%20%20%22city%22%3A%20%22%22%2C%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20//%20city%2C%20capitalised%0A%20%20%20%20%22social_media%22%3A%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20//%20social%20media%20links%0A%20%20%20%20%20%20%22meetup%22%3A%20%22rladies-%22%2C%20%0A%20%20%20%20%20%20%22twitter%22%3A%20%22%22%2C%0A%20%20%20%20%20%20%22email%22%3A%20%22%22%0A%20%20%20%20%7D%2C%0A%20%20%20%20%22organizers%22%3A%20%7B%0A%20%20%20%20%20%20%22current%22%3A%20%5B%5D%2C%20//%20comma%20separated%20and%20each%20item%20quoted%0A%20%20%20%20%20%20%22former%22%3A%20%5B%5D%0A%20%20%20%20%7D%0A%20%20%7D%0A%5D). This link will fork the repository to your user account, and initiate a new file with some template content in it.
+
+## File name
```

Comment:
I guess this is automatically generated. I'd recommend adding an empty line after the header. I don't remember why though.